### DISCUSSION TITLE
compare profiles

### DISCUSSION THREAD

**Discussion by CodyPrince** [OrcaSlicer/OrcaSlicer#1056] [Q&A]
Is there a way to compare between orcaslicer saved profiles like Prusaslicer and Superslicer? I'm moving most of my printing to ocraslicer and just realized this feature is either missing or hiding. 1.6.3 beta

Thanks, Cody

**Comment by hliebscher** [comment]
only manual  but if you miss a feature in Orca make a feature request.  if this feature is in Prusa or SuperSlicer it is possible to port to the OS
